larpandoracontent/LArHelpers/LArMonitoringHelper.h

```diff
@@ -75,6 +75,46 @@ class LArMonitoringHelper
      */
     static void PrintMatchingTable(const pandora::PfoVector &orderedPfoVector, const pandora::MCParticleVector &orderedMCParticleVector,
         const LArMCParticleHelper::MCParticleToPfoHitSharingMap &mcParticleToPfoHitSharingMap, const unsigned int nMatches);
+
+    // -- Rand index helpers
+
+    /**
+     *  @brief  Calculate the adjusted Rand Index for a given contingency table that summarises two clusterings A and B.
+     *          Adjusted Rand Index is a measure of the similarity of two clusterings that is bounded above by 1 (identical)
+     *          and is 0 when the two clusterings are as similar as two random clusterings with the same number of clusters.
```

Comment:
Probably worth noting the lack of a lower bound (unless we enforce one), for the "worse than random" case.